src/components/header/HeaderNetStatus.tsx

```diff
@@ -90,17 +91,64 @@ const HeaderNetStatus: React.FC<Props> = (_: Props): JSX.Element => {
     )
   }, [menuItems, theme])
 
+  type RootItemProps = { children: React.ReactNode }
+
+  const menuMobile = menuItems.map((item, i) => {
+    const { label, key, url } = item
+    const color = palette('text', 0)({ theme })
+    const RootElement = (props: RootItemProps) =>
+      i === menuItems.length - 1 ? (
```

Comment:
It might not needed if you just use `HeaderDrawerItem` (see my comment in `HeaderStyle`)